src/Utils/search-products.js

```diff
@@ -0,0 +1,4 @@
+export const searchProducts = (products, value) =>
+	products.filter((product) =>
+		product.title.toLowerCase().includes(value.toLowerCase()) ? product : null
+	);
```

Comment:
```javascript
export const searchProducts = (products, value) =>
  products.filter((product) =>
    product.title.toLowerCase().includes(value.toLowerCase())
  );
```
`includes()` methods return true or false, so if you want you want you can trim the fat!

Reply:
Thank you for pointing that out, I will remove the unnecessary code